packages/block-library/src/latest-posts/edit.js

```diff
@@ -547,9 +547,20 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 							{ displayPostContent &&
 								displayPostContentRadio === 'excerpt' && (
 									<div className="wp-block-latest-posts__post-excerpt">
-										{ postExcerpt }
+										{  excerpt
+											.trim()
+											.split( ' ', excerptLength )
+											.join( ' ' ) }
+										{ /* translators: excerpt truncation character, default …  */ }
+										<a
+											href={ post.link }
+											rel="noopener noreferrer"
+											onClick={ showRedirectionPreventedNotice }
```

Comment:
This link should contain context with the post title. You can use an `aria-label` for this.

```
sprintf( __( 'Read more about %s', post_title_var ) );
```

Reply:
Added